sweep/sweeper.go

```diff
@@ -719,6 +724,8 @@ func (s *UtxoSweeper) scheduleSweep(currentHeight int32) error {
 	startTimer := false
 	for _, cluster := range s.clusterBySweepFeeRate() {
 		// Examine pending inputs and try to construct lists of inputs.
+		// We don't need to obtain the coin selection lock, because
+		// those coins may not be available when we sweep anyway.
```

Comment:
May not be available because they were used in another transaction?

Reply:
Rewritten comment. The basic idea is that we are just after an indication here